registry.json

```diff
@@ -1697,6 +1697,7 @@
   {
     "id": "xdai",
     "name": "xDai",
+    "displayName": "Gnosis Chain",
     "coinId": 10000100,
     "symbol": "xDAI",
```

Comment:
symbol needs to be changed too?

Reply:
https://www.xdaichain.com/for-users/getting-started-with-gc, still `xDAI`

Reply:
👍